understanding/20/use-of-color.html

```diff
@@ -157,18 +170,27 @@ <h2>Examples of Use of Color</h2>
       <dl>
          <dt>A form that uses color and text to indicate required fields</dt>
          <dd>A form contains both required and optional fields. Instructions at the top of the
-               form explain that required fields are labeled with red text and also with an icon.
-               Users who cannot perceive the difference between the optional field labels and the
-               red labels for the required fields will still be able to see the icon next to the
-               red labels.</dd>
+             form explain that required fields are labeled with red text and also with an icon.
+             Users who cannot perceive the difference between the optional field labels and the
+             red labels for the required fields will still be able to see the icon next to the
+             red labels.</dd>             
          <dt>An examination</dt>
          <dd>Students view an SVG image of a chemical compound and identify the chemical elements
-               present based <strong>both</strong> on the colors used, as well as numbers next to each
-               element. A legend shows the color and number for each type of element. Sighted users who
-               cannot perceive all the color differences can still understand the image by relying on
-               the numbers.</dd>
+             present based <strong>both</strong> on the colors used, as well as numbers next to each
+             element. A legend shows the color and number for each type of element. Sighted users who
+             cannot perceive all the color differences can still understand the image by relying on
+             the numbers.</dd>
+         <dt>A series of toggle buttons</dt>
```

Comment:
again, non-text contrast can't compare things that aren't visually adjacent. use of color, on the other hand, can

Reply:
the point of this example was the "also styled to appear pressed" and the alternative use of icon, just to make sure it's more than just a change of background colour alone (below 3:1) that distinguishes toggles that are on and toggles that are off. without those, you're relying purely on the user's ability to be able to tell the difference between the two shades, which fails use of color (and *can't* fail non-text contrast because they're not adjacent, unless the toggles all butt up with each other with no dividing line or anything)